### DISCUSSION TITLE
Not found node_modules/@angular/common/bundles/common-http.umd.js

### DISCUSSION THREAD

**Discussion by Shilpa1TR** [webpack/webpack#18248] [Q&A]
I am using angular 16 and i want to bundle the node modules library hence i am using System config to map.

In angular 16 i could see .umd.js file is not available. So can i use .mjs extension instead .umd.js.

node_modules/@angular/core/bundles/core.umd.js

replace with

node_modules/@angular/core/esm2022/src/core.mjs

Please let us know.

**Comment by alexander-akait** [comment]
Sorry, your information is not enough to answer, please provide a reproducible example, you can use any extension
